### DISCUSSION TITLE
How to modify dashboard ?

### DISCUSSION THREAD

**Discussion by vvince62550** [DigiLive/mushroom-strategy#117] [Q&A] [answered]
Hello,
When my dashboard is generated, I can only modify one time !
After I have a message
"No visual editor available for type "custom:mushroom-strategy"."

Could yoou help me ?

Thanks,

Vincent

**Comment by DigiLive** [comment, marked as answer]
There's no visual editor, just a yaml (raw configuration) editor.

Did you follow the guide at https://github.com/AalianKhan/mushroom-strategy/wiki?

Seems like the content of your yaml is incorrect.

**Comment by vvince62550** [comment]
Now it's working, I have possibility to change position, edit code of blocks,...

**Comment by clowgg** [comment]
Now I have the basic editing working (_phew_), I have a few questions:

1) Is there a way to set 3 columns of icons for the Areas?
2) Can you centre the 'Areas' title, and add a sub-title?
3) Is there a way to show what zone the people are in under their avatars?
4) Can we control the spacing of the Chips in the header?

**Comment by DigiLive** [comment]
> Now I have the basic editing working (_phew_), I have a few questions:
> 
>     1. Is there a way to set 3 columns of icons for the Areas?
> 
>     2. Can you centre the 'Areas' title, and add a sub-title?
> 
>     3. Is there a way to show what zone the people are in under their avatars?
> 
>     4. Can we control the spacing of the Chips in the header?

1. Not yet. Follow #84.
2. AFAIK, Mushroom cards doesn't support alignment. It's does support a sub-title, but that's not implemented in our code. You can request it by creating a new issue.
3. You can request it by creating a new issue.
4. AFAIK, Mushroom cards doesn't support spacing.

**Comment by DigiLive** [comment]
This discussion is spreading out into too much topics, therefor I'm closing it.
Please try to limit a discussion to a single topic/question.
